.github/actions/javascript/authorChecklist/authorChecklist.ts

```diff
@@ -54,7 +54,7 @@ function partitionWithChecklist(body: string): string[] {
 async function getNumberOfItemsFromAuthorChecklist(): Promise<number> {
     const response = await fetch(pathToAuthorChecklist);
     const fileContents = await response.text();
-    const checklist = partitionWithChecklist(fileContents)[1];
+    const checklist = partitionWithChecklist(fileContents).at(1) ?? '';
```

Comment:
I see a lot of `?? ''` usage across this PR, I think our standard is to default to `'-1'`, copying from the guidelines:
```ts
### Default value for inexistent IDs

 Use `'-1'` or `-1` when there is a possibility that the ID property of an Onyx value could be `null` or `undefined`.

``` ts
// BAD
const foo = report?.reportID ?? '';
const bar = report?.reportID ?? '0';

report ? report.reportID : '0';
report ? report.reportID : '';

// GOOD
const foo = report?.reportID ?? '-1';

report ? report.reportID : '-1';
```